Added rigid rotation model to `diff_rot()` (#5132)
sunpy/sunpy: changelog/5132.feature.rst

> Added Rigid Rotation Model to :func:`~sunpy.physics.differential_rotation.diff_rot`, which can be used by passing `rot_type` as `"rigid"`.

Reviewer: ```suggestion
Added Rigid Rotation Model to :func:`~sunpy.physics.differential_rotation.diff_rot`, which can be used by passing ``rot_type`` as "rigid".
```

> Added Rigid Rotation Model to :func:`~sunpy.physics.differential_rotation.diff_rot`, which can be used by passing ``rot_type`` as "rigid".

Reviewer 2: ```suggestion
Added a rigid rotation model to :func:`~sunpy.physics.differential_rotation.diff_rot` via ``rot_type=rigid``, where the rotation rate does not vary with latitude.
```
